cmake/cvars.cmake

```
@@ -0,0 +1,50 @@
+set(CVAR_VSYNC_ENABLED "gVsyncEnabled" CACHE STRING "")
+set(CVAR_Z_FIGHTING_MODE "gZFightingMode" CACHE STRING "")
+set(CVAR_NEW_FILE_DROPPED "gNewFileDropped" CACHE STRING "")
+set(CVAR_DROPPED_FILE "gDroppedFile" CACHE STRING "")
+set(CVAR_INTERNAL_RESOLUTION "gInternalResolution" CACHE STRING "")
+set(CVAR_MSAA_VALUE "gMSAAValue" CACHE STRING "")
+set(CVAR_SDL_WINDOWED_FULLSCREEN "gSdlWindowedFullscreen" CACHE STRING "")
+set(CVAR_TEXTURE_FILTER "gTextureFilter" CACHE STRING "")
+set(CVAR_CONTROL_NAV "gControlNav" CACHE STRING "")
```

we should probably make this

```suggestion
set(CVAR_IMGUI_CONTROLLER_NAV "gControlNav" CACHE STRING "")
```

or something else that implies it's an ImGui related thing